lib/fetchCharacters.js

```diff
@@ -0,0 +1,15 @@
+export default async function fetchCharacters() {
+  const response = await fetch("https://rickandmortyapi.com/api/character/");
+  const body = await response.json();
+  const characters = body.results;
+  return characters;
+}
+
+export async function fetchRandomCharacters(random) {
```

Comment:
Those don't seem to be random, a more fitting name would be 
```suggestion
export async function fetchMultipleCharacters(characterIds) {
```